10.1/Dockerfile.fedora

```diff
@@ -19,14 +19,12 @@ LABEL summary="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.description="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MariaDB 10.1" \
       io.openshift.expose-services="3306:mysql" \
-      io.openshift.tags="database,mysql,mariadb,mariadb101,galera"
-
-ENV NAME=mariadb VERSION=10.1 RELEASE=1 ARCH=x86_64
```

Comment:
I think removing the ENV definition will make the LABELS defined empty.. I also kind of like the idea of having environment variables like those defined, because:
* we can re-use them in the labels specification (e.g. `version="$VERSION"`)
* they will be available in the image (so, we might use them in the scripts), labels are not

So, maybe we can define them like this in RHEL/CentOS Dockerfiles as well.. The variables should be defined on separate lines though, like:
```
ENV NAME="mariadb" \
    VERSION="10.1" \
    RELEASE="1" \
    ARCH="x86_64"
```